stopcovid/dialog/engine.py

```diff
@@ -265,6 +267,15 @@ def _check_response(
 
         return events
 
+    def _drill_requested(
+        self, dialog_state: DialogState, base_args: Dict[str, Any]
+    ) -> Optional[List[DialogEvent]]:
+        prompt = dialog_state.get_prompt()
+        if prompt is None:
```

Comment:
Any reason we're checking `get_prompt()` rather than `dialog_state.current_drill`? I'm not sure if theres any meaningful difference, but curious to hear your thoughts

Reply:
good question--I was just going with the pattern from`_next_drill_requested`, but I do think that checking `current_drill` makes the intent more clear